apps/wallet-mobile/src/features/Send/useCases/StartMultiTokenTx/SelectNameServer/SelectNameServer.tsx

```diff
@@ -51,7 +51,7 @@ export const SelectNameServer = () => {
 
           <Spacer height={4} />
 
-          <ButtonGroup labels={labels} onSelect={handleOnSelectNameServer} />
+          <ButtonGroup initial={0} labels={labels} onSelect={handleOnSelectNameServer} />
```

Comment:
shouldn't the initial be the `labels.indexOf(target.receiver.selectedNameServer)` or is it the order guaranteed?